packages/agent/conf/demos/knb/auckland/oid4vci_metadata/knb_van_auckland.json

```diff
@@ -130,7 +130,7 @@
             ]
           }
         },
-        "id": "knb-notary",
+        "id": "VanAuckland2023",
```

Comment:
i think niels mentioned that we should not add any years. just checking that here

Reply:
fixed